doc/PostProcessing.md

```diff
@@ -389,6 +389,8 @@ use the boolean flag `"producePlotsOnUpdate"`.
 
 To pick up the last existing trend which matches the specified Activity, set `"resumeTrend"` to `"true"`.
 
+To generate plots only when all input objects are available, set `"trendOnlyIfAllInputsAreAvailable"`.
```

Comment:
I am wondering if we could make the parameter shorter... Perhaps "trendIfAllInputs"?